xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -166,8 +166,8 @@ The examined data communicates to the pipeline when it should signal more data i
       </Parameters>
       <Docs>
         <param name="leaveOpen">An optional flag that indicates whether disposing the returned <see cref="T:System.IO.Stream" /> leaves <see cref="T:System.IO.Pipelines.PipeReader" /> open (<see langword="true" />) or completes <see cref="T:System.IO.Pipelines.PipeReader" /> (<see langword="false" />).</param>
-        <summary>Returns the underlying <see cref="T:System.IO.Stream" /> wrapped by the <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
-        <returns>The underlying stream.</returns>
+        <summary>Returns a <see cref="T:System.IO.Stream" /> which wraps the <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
```

Comment:
While the current version is wrong, PipeReader is abstract and AsStream is virtual. The default implementation does just wrap the PipeReader, but this could conceivably be different for a PipeReader that started out wrapping a Stream for example. I would just make the summary more generic.

```suggestion
        <summary>Returns a <see cref="T:System.IO.Stream" /> representation of the <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
```

Reply:
Good suggestion, thanks